docs/ldml/tr35-keyboards.md

```diff
@@ -2491,7 +2494,7 @@ TODO: Rewrite this? Probably push out to each element's section?
 3.  No `keyMap[@modifiers]` value can overlap with another `keyMap[@modifiers]` value.
```

Comment:
Just realised this section is all commented out and wrong anyway (e.g. there is no `<map` element. It's a TODO that should be resolved before we publish.

Reply:
I'm going to remove the TODO as part of the general cleanup ticket.